libbeat/cmd/instance/metrics.go

```diff
@@ -149,7 +149,7 @@ func getCPUUsage() (float64, *process.Ticks, error) {
 
 	totalCPUUsage, ok := iTotalCPUUsage.(float64)
 	if !ok {
-		return 0.0, nil, fmt.Errorf("error converting value of CPU usage since start")
+		return 0.0, nil, fmt.Errorf("error converting value of CPU usage since start to float64")
 	}
```

Comment:
Adding the original value of `totalCPUUsage` to the message would be useful when debugging. Same with the other casting for this PR.